staging/src/k8s.io/component-base/metrics/legacyregistry/registry.go

```diff
@@ -42,6 +42,9 @@ var (
 
 	// Register registers a collectable metric but uses the global registry
 	Register = defaultRegistry.Register
+
+	// Registerer exposes the global registerer
+	Registerer = defaultRegistry.Registerer
```

Comment:
it looks like use of this (rather than calling `Register`) skips calling `Create` on the passed-in metric, skips calling `addResettable`, and skips calling `trackHiddenCollector`... are any of those things problematic?

Reply:
No, for external/library metrics, they are exempt from stability classes.

Reply:
I don't know these details; I'm hoping @logicalhan can answer this